Support for Nextjs 11.

Wondering if possible to add support for this?

**Describe the bug**

Support required for Nextjs 11.

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! node_modules/next
npm ERR!   next@"11.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer next@">=9.5.0 <11" from next-firebase-auth@0.13.1
npm ERR! node_modules/next-firebase-auth
npm ERR!   next-firebase-auth@"^0.13.1" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\krthush\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\krthush\AppData\Local\npm-cache\_logs\2021-06-18T16_27_36_936Z-debug.log
```

**Version**
^0.13.1

--- a/package.json
+++ b/package.json
@@ -81,7 +81,7 @@
   "peerDependencies": {
     "firebase": ">=7.0.0 <9",
     "firebase-admin": "^9.0.0",
-    "next": ">=9.5.0 <11",
+    "next": ">=9.5.0 <12",
     "react": ">=16.8.0 <18",
     "react-dom": ">=16.8.0 <18"
   },

